### PR TITLE
add custom rotater

### PR DESCRIPTION
**Pull request by stegben**
#25

### CODE REVIEW

**Comment by tw0517tw** [comment]
any document about it?

**Comment by chentsulin** [comment]
https://github.com/jasondavies/d3-cloud#rotate

**Comment by stegben** [comment]
Its document is not complete. One can actually pass words to the function

File: src/WordCloud.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import cloud from 'd3-cloud';
 const fill = d3.scaleOrdinal(d3.schemeCategory10);
 
 const defaultFontSizeMapper = word => word.value;
+const defaultRotate = word => word.value % 180 - 90; // eslint-disable-line no-mixed-operators
```
Comment (stegben) [inline src/WordCloud.js]: 括號在這裡看起來好多餘...

Comment (stegben) [inline src/WordCloud.js]: fixed

**Pull request merged**